packages/deploy/deploy/800_oft_sand/802_oft_adapter_setup_sepolia.ts

```diff
@@ -0,0 +1,36 @@
+import {DeployFunction} from 'hardhat-deploy/types';
```

Comment:
I'm still confused by the naming here. The file and tags refer to `setup_sepolia`, yet the functions refer to baseSepolia and bscTestnet?

Reply:
Same for the other files - perhaps some dev notes would be helpful ?

Reply:
sure, will add those